web3.js/src/transaction.ts

```diff
@@ -133,11 +149,38 @@ export type SignaturePubkeyPair = {
 
 /**
  * List of Transaction object fields that may be initialized at construction
- *
  */
 export type TransactionCtorFields = {
+  /** Optional nonce information used for offline nonce'd transactions */
+  nonceInfo?: NonceInformation | null;
+  /** The transaction fee payer */
+  feePayer?: PublicKey | null;
+  /** One or more signatures */
+  signatures?: Array<SignaturePubkeyPair>;
+  /** A recent blockhash */
+  recentBlockhash?: Blockhash;
+};
+
+/**
+ * List of Transaction object fields that may be initialized at construction
+ */
+export type TransactionBlockhashCtor = {
+  /** The transaction fee payer */
+  feePayer?: PublicKey | null;
+  /** One or more signatures */
+  signatures?: Array<SignaturePubkeyPair>;
   /** A recent blockhash */
-  recentBlockhash?: Blockhash | null;
+  latestBlockhash?: {
```

Comment:
Why optional. Aren't we saying that if you want to call this new modern method, you _have_ to bring one of these?